src/app/scripts/cac/pages/cac-pages-home.js

```diff
@@ -1,72 +1,81 @@
 CAC.Pages.Home = (function ($, UserPreferences) {
     'use strict';
 
-    var defaults = {};
+    var defaults = {
+        selectors: {
+            directionsForm: '#directions',
+            directionsFrom: '#directionsFrom',
+            directionsMode: '#directionsMode',
+            directionsTo: '#directionsTo',
+            exploreForm: '#explore',
+            exploreMode: '#exploreMode',
+            exploreOrigin: '#exploreOrigin',
+            exploreTime: '#exploreTime',
+            toggleButton: '.toggle-search button',
+            typeahead: 'input.typeahead'
+        }
+    };
+    var options = {};
 
-    function Home(options) {
-        this.options = $.extend({}, defaults, options);
+    function Home(params) {
+        options = $.extend({}, defaults, params);
     }
 
     Home.prototype.initialize = function () {
-        $('.toggle-search button').on('click', function(){
+        $(options.selectors.toggleButton).on('click', function(){
             var id = $(this).attr('id');
-
-            if (id === 'toggle-directions') {
-                $('#explore').addClass('hidden');
-                $('#directions').removeClass('hidden');
-            } else {
-                $('#directions').addClass('hidden');
-                $('#explore').removeClass('hidden');
-            }
+            setTab(id);
         });
 
-        this.typeahead = new CAC.Search.Typeahead('input.typeahead');
+        this.typeahead = new CAC.Search.Typeahead(options.selectors.typeahead);
         this.typeahead.events.on('cac:typeahead:selected', $.proxy(onTypeaheadSelected, this));
-    };
 
-    // save form data and redirect to map when 'go' button clicked
+        // save form data and redirect to map when 'go' button clicked
 
-    $('#explore').submit(function(event) {
-        event.preventDefault();
-        var exploreTime = $('#exploreTime').val();
-        var mode = $('#exploreMode').val();
-        var originText = $('#exploreOrigin').val();
+        $(options.selectors.exploreForm).submit(function(event) {
```

Comment:
Might want to take these two event handler functions and move them to named functions.